### PR TITLE
Implemented unittests for dc2deployment

### PR DESCRIPTION
**Pull request by drajshek**

### CODE REVIEW

File: pkg/dc2deployment/dc2deployment_test.go [hunk 1]
```diff
@@ -0,0 +1,94 @@
+package dc2deployment
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"path/filepath"
+	"testing"
+
+	dcAPI "github.com/openshift/api/apps/v1"
```
Comment (chrisjs) [inline pkg/dc2deployment/dc2deployment_test.go]: only suggestion id have is changing `dcAPI` to `apps`. looks good otherwise!

**Review by codegold79 — COMMENTED** [review]
Besides what @chrisjs suggested, I can't think of anything else. These tests are awesome, and written very quickly. Well done!

File: pkg/dc2deployment/dc2deployment_test.go (hunk 1 above)
Comment (drajshek) [reply pkg/dc2deployment/dc2deployment_test.go]: implemented the review comments.

**Review by codegold79 — APPROVED** [review]
lgtm

**Pull request merged**